lib/registries/addon/components/registration-form-view/x-options/styles.scss

```diff
@@ -0,0 +1,23 @@
+.OptionsList {
+    list-style: none;
+    margin-left: 0;
+    padding-left: 40px;
+    text-indent: -20px;
+}
+  
+.OptionsList li::before {
+    content: '✓';
+    padding-right: 8px;
+}
+
+.OptionsList li { 
+    padding: 5px 0;
+}
+
+.OptionsList li:first-of-type {
+    padding-top: 0;
+}
+
+.OptionsList li:last-of-type {
```

Comment:
Minor nit: This would be a good place to use Sass's nested selectors:
```scss
.OptionsList {
  // ...

  li::before { }

  li { }

  li:first-of-type { }

  li:last-of-type { }
}
```
Just [remember not to nest too deep](http://thesassway.com/beginner/the-inception-rule).